cmd/installer/kotscli/kotscli.go

```diff
@@ -53,12 +55,18 @@ func Install(opts InstallOptions) error {
 		upstreamURI = fmt.Sprintf("%s/%s", upstreamURI, channelSlug)
 	}
 
+	licenseFile, err := writeLicenseToTempFile(opts.License)
```

Comment:
kots expects the license file to be as it is downloaded from the vendor portal. unmarshaling / remarshalling can introduce bugs and regressions when the license type gets updated that may lead to invalid signatures. i think we should keep the old logic of passing the license file directly to kots.